bigbluebutton-tests/playwright/user/multiusers.js

```diff
@@ -267,6 +267,37 @@ class MultiUsers {
     await this.modPage.hasElement(e.multiUsersWhiteboardOn);
   }
 
+  async removeUser() {
+    await this.modPage.waitAndClick(e.userListItem);
+    await this.modPage.waitAndClick(e.removeUser);
+    await this.modPage.waitAndClick(e.removeUserConfirmationBtn);
+
+    //Will be modified when the issue is fixed and accept just one of both screens
+    //https://github.com/bigbluebutton/bigbluebutton/issues/16463
+    try {
+      await this.modPage2.hasElement(e.errorScreenMessage);
+    } catch (err) {
+      await this.modPage2.hasElement(e.meetingEndedModalTitle);
+    }
+  }
+
+  async removeUserAndPreventRejoining(context) {
+    await this.modPage.waitAndClick(e.userListItem);
+    await this.modPage.waitAndClick(e.removeUser);
+    await this.modPage.waitAndClick(e.confirmationCheckbox);
+    await this.modPage.waitAndClick(e.removeUserConfirmationBtn);
+
+    //Will be modified when the issue is fixed and accept just one of both screens
+    //https://github.com/bigbluebutton/bigbluebutton/issues/16463
+    try {
+      await this.modPage2.hasElement(e.errorScreenMessage);
+    } catch (err) {
+      await this.modPage2.hasElement(e.meetingEndedModalTitle);
+    }
+    
+    await this.initModPage2(false, context, {meetingId: this.modPage.meetingId, customParameter: 'userID=Moderator2'})
+    await this.modPage2.hasText(e.userBannedMessage, /banned/);
+
```

Comment:
The closing semicolon is missing
```suggestion
}

```